### ISSUE TITLE
Error: ImageInput::create() called with no filename

### ISSUE THREAD

**Issue by ddesmond** [RenderKit/oidn#228]
Hi!

I have managed to setup the toolset to compile oidin on windows10. I can build the solution without error in vs 2022, the oidnTest.exe works, but when i try to load a jpg or exr i got this error:
```
OpenImageIO exited with a pending error message that was never
retrieved via OIIO::geterror(). This was the error message:
ImageInput::create() called with no filename
```

my full command would be:
`oidnDenoise.exe --ldr C:\Users\compiler\mazda_64spp_input.jpg -o C:\Users\compiler\mazda_64spp_input_denoised.jpg`

I did pull all from github and setup TBB and ISPCI and passed all possible errors with opeimageio.
At the end what works is to use cmake-gui and config then generate then build in vs2022.
I do use vcpkg and oiio is from there.

Any clues?

thanks!

**Comment by ddesmond** [comment]
I have tried to rebuild with the tag 1.4.2 and same openimageio from vcpkg and this compiles and i can load exrs and save out.

I suppose the issue is in newer builds and missing argument to the create imagebuffer or so when passing to oiio.

Which version tag above 2.0 would you suggest to build and have oiio working?

thanks!

**Comment by ddesmond** [comment]
closing since no one reposnded and the issue is not resolved yet.
marry xmass!

**Comment by atafra** [comment]
Sorry for the delay.

I cannot reproduce the issue. I used the latest release and tried two different versions of OpenImageIO from `vcpkg`: 2.3.17 and the latest 3.0.1 (which requires a small fix: https://github.com/RenderKit/oidn/commit/4f5751a530138fe1e9904c039c76eb3f2aff7b7d).

I used the same command line arguments as you, except the path was different. I don't think this is a bug in OIDN. It's perhaps an OpenImageIO setup issue on your machine (does the OpenImageIO DLL in your path match the version you're building with?).

Happy holidays!
